### PR TITLE
Centered Initialization of Forms and Consistent Element Positioning

### PR DESCRIPTION
**Pull request by DanielSvoboda**
Now, all forms are initialized centered on the screen.
WindowStartupLocation="CenterOwner">

Adjustments have been made to ensure consistent
element positioning, providing a more organized
and user-friendly experience in:
MainWindow.axaml
InfoWindow.axaml

See the screenshot below:
print

print 2

### CODE REVIEW

File: UABEAvalonia/Forms/MainWindow.axaml [hunk 1]
```diff
@@ -4,10 +4,13 @@
         xmlns:mc="http://schemas.openxmlformats.org/markup-compatibility/2006"
         mc:Ignorable="d" d:DesignWidth="400" d:DesignHeight="150"
         Width="400" Height="150"
+        MinWidth="400" MinHeight="150"
```
Comment (nesrak1) [inline UABEAvalonia/Forms/MainWindow.axaml:7]: Sometimes there can be really long names in bundles, so I'd rather not have this set at a static width.

Comment (DanielSvoboda) [reply UABEAvalonia/Forms/MainWindow.axaml]: it makes sense, I removed the 2 lines

**Comment by nesrak1** [comment]
I can look over this in a few days and accept it but I don't know if it'll make it into a new UABEA (non-docking) release before the big update.

**Comment by DanielSvoboda** [comment]
Ok, fine, I'll just click on review because in a few days it will be 1 year since this PR was opened.

**Comment by eriajohnpaul12-cyber** [comment]
Why my tools doesn't have import obj or on the plugins when highlights the mesh it's doesn't have import obj or export obj

**Comment by nesrak1** [comment]
@eriajohnpaul12-cyber why ask in this PR? This PR has nothing to do with mesh import or export. This has been requested many times: #60, #111, #124, among others. It's difficult to do - both from the user perspective and the developer perspective, especially without breaking anything. So I don't currently support it.

**Pull request merged**